tests/unit/components/Editor/ItemEditor.spec.js

```diff
@@ -27,6 +27,12 @@ describe("ItemEditor.vue", () => {
       options: ["", "", ""],
       correct_answer: [0, 1],
     },
+    {
+      text: "test",
+      type: "mcq",
+      options: ["", ""],
+      survey: true,
+    },
```

Comment:
You added one more array element to `itemDetailList`. So there are 3 elements in `itemDetailList` but only 2 in `itemList`. This is not logically correct! I don't know who the tests are not failing lol